### PR TITLE
Hunterize v1.12.2-8d6df7b

### PR DESCRIPTION
**Pull request by ned14**
cmake hunter packaging plus latest updates to ASIO v1.12 stable branch.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,97 @@
+* text=auto !eol svneol=native#text/plain
```
Comment (rbsheth) [inline .gitattributes:1]: I don't think we need this file checked in here, right?

**Review by rbsheth — COMMENTED** [review]
@ned14 Thanks for your PR! I found the branch name confusing - I think I changed the base of your PR to the latest 1.12 (not 1.14) commit, can you double-check?

File: .gitattributes (hunk 1 above)
Comment (ned14) [reply .gitattributes:1]: I'll be contributing said file back to ASIO upstream. I'm fairly sure Chris will accept that bit.

**Comment by ned14** [comment]
Yes the branch naming is a legacy artifact. It was always the 12.2 stable branch.

File: .gitattributes (hunk 1 above)
Comment (ned14) [reply .gitattributes:1]: https://github.com/chriskohlhoff/asio/pull/444

Comment (rbsheth) [reply .gitattributes:1]: Okay, we'll make sure to rebase when updating.

**Review by rbsheth — APPROVED** [review]

**Comment by rbsheth** [comment]
@ned14 Released: https://github.com/cpp-pm/asio/archive/v1.12.2-8d6df7b-p0.tar.gz

**Comment by ned14** [comment]
Cool, thanks.

**Pull request merged**